src/main/java/de/tum/cit/aet/artemis/lecture/domain/LectureUnit.java

```diff
@@ -78,6 +79,10 @@ public abstract class LectureUnit extends DomainObject implements LearningObject
     @JsonIgnore // important, so that the completion status of other users do not leak to anyone
     private Set<LectureUnitCompletion> completedUsers = new HashSet<>();
 
+    @OneToOne(mappedBy = "lectureUnit", fetch = FetchType.LAZY)
```

Comment:
sorry for my earlier comment, the fetch type has no effect on the parent side of the relation and can be removed.